Deploy branch

NelsonAraujo93 opened the pull request:
In this pull i followed the next steps:

- Read the deployment documentation.
- Create a new branch.
- Check Linters.
- Create the GH page.
- Check responsiveness.

Commit:
Create CNAME

Commit:
Delete CNAME

Commit:
Merge branch 'main' into deploy-branch

pinkmoon25 reviewed (changes requested):
Hi @NelsonAraujo93 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

<img src="https://media1.giphy.com/media/KyBesf8kNT6s88SEgb/giphy.gif"/>

## Highlights
linter checks :heavy_check_mark: 
Professional Readme :heavy_check_mark: 

# Required Changes ♻️

### - [ ] Hi @NelsonAraujo93 , Please deploy your portfolio with github pages, you have to deploy your portfolio live you can check how to do that [here](https://docs.github.com/en/pages/getting-started-with-github-pages/creating-a-github-pages-site) this will create a github pages site for your portfolio, then please use that link and add it to the live demo section of your readme and ask for a re-review. let me know if you have trouble understanding anything from the article above Thanks :pray: 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Commit:
Update README.md

As the first reviewer asked for i updated the Read.me file with the Live demo link

brainconnect93 reviewed (approved):
## Status: Approved :heavy_check_mark:

Hi @NelsonAraujo93, 

Good Job Nelson, You have a really fantastic portfolio I like it.
Your project is complete! Great job for following the project requirement🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 


<img src="https://camo.githubusercontent.com/4799731d86dd0d52a0bb7e22af48077bd16556ee26061146e987e9a98d36ee0d/68747470733a2f2f6d65646961302e67697068792e636f6d2f6d656469612f594b646d3931584f4e4c5461772f67697068792d646f776e73697a65642d6d656469756d2e676966">

## Good Points 👍

- Has a proper Pull Request. ✔️
- No Linter errors ✔️
- Good use of Github flow ✔️
- Deployed with Github page ✔️

## Optional changes

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brainconnect93 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
